clio-client/src/main/scala/org/broadinstitute/clio/client/webclient/ClioWebClient.scala

```diff
@@ -59,101 +56,51 @@ class ClioWebClient(
     dispatchRequest(HttpRequest(uri = "/health"))
   }
 
-  def getSchemaWgsUbam(
-    implicit bearerToken: OAuth2BearerToken
-  ): Future[HttpResponse] = {
+  def getSchema(
+    transferIndex: TransferIndex
+  )(implicit credentials: HttpCredentials): Future[HttpResponse] = {
     dispatchRequest(
-      HttpRequest(uri = "/api/v1/wgsubam/schema")
-        .addHeader(Authorization(credentials = bearerToken))
+      HttpRequest(uri = s"/api/v1/${transferIndex.urlSegment}/schema")
```

Comment:
Not in scope for this review but at some point we will want to pass in version.

Reply:
Yeah I'm not 100% sure how versioning will work, we may need different types for input/query/output if we change the schema in an incompatible way.